<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela_Principal</class>
 <widget class="QMainWindow" name="Janela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="botao_cadastro">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 10px 10px 20px 20px;
	background-color: rgb(0, 170, 255);
}
QPushButton:hover{
border-radius: 10px 10px 20px 20px;
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_acessar_conta">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 10px 10px 20px 20px;
	background-color: rgb(0, 170, 255);
}
QPushButton:hover{
border-radius: 10px 10px 20px 20px;
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_remedios">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 10px 10px 20px 20px;
	background-color: rgb(0, 170, 255);
}
QPushButton:hover{
border-radius: 10px 10px 20px 20px;
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Procurar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao_carrinho">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
border-radius: 10px 10px 20px 20px;
	background-color: rgb(0, 170, 255);
}
QPushButton:hover{
border-radius: 10px 10px 20px 20px;
	background-color: rgb(85, 170, 255);
}
</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>-1</y>
      <width>431</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 87 20pt &quot;Arial Black&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Farm√°cia Popular&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>231</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Imagens/farmacia.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Imagens/farmacia (1).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
